deps/media-playback/media-playback/closest-format.h

```diff
@@ -38,23 +38,23 @@ static enum AVPixelFormat closest_format(enum AVPixelFormat fmt)
 		return AV_PIX_FMT_YUV444P;
 
 	case AV_PIX_FMT_YUV422P:
-		return AV_PIX_FMT_YUV422P;
-
-	case AV_PIX_FMT_YUV422P10LE:
-		return AV_PIX_FMT_YUV422P10LE;
-
 	case AV_PIX_FMT_YUVJ422P:
-	case AV_PIX_FMT_UYVY422:
 	case AV_PIX_FMT_YUV422P16LE:
 	case AV_PIX_FMT_YUV422P16BE:
 	case AV_PIX_FMT_YUV422P10BE:
 	case AV_PIX_FMT_YUV422P9BE:
 	case AV_PIX_FMT_YUV422P9LE:
-	case AV_PIX_FMT_YVYU422:
 	case AV_PIX_FMT_YUV422P12BE:
 	case AV_PIX_FMT_YUV422P12LE:
 	case AV_PIX_FMT_YUV422P14BE:
 	case AV_PIX_FMT_YUV422P14LE:
+		return AV_PIX_FMT_YUV422P;
+
+	case AV_PIX_FMT_YUV422P10LE:
+		return AV_PIX_FMT_YUV422P10LE;
+
+	case AV_PIX_FMT_UYVY422:
+	case AV_PIX_FMT_YVYU422:
```

Comment:
Return `AV_PIX_FMT_YVYU422` for `AV_PIX_FMT_YVYU422` since OBS has VIDEO_FORMAT_YVYU.

Reply:
This requires fixing obs_to_ffmpeg_video_format to support VIDEO_FORMAT_YVYU

Reply:
Also delete ffmpeg_to_obs_video_format while you're there. I'm pretty sure that's a dead function.

Reply:
Also need to update ffmpeg-decode.c and media.c, heh. Sorry about this.

Reply:
OBS added VIDEO_FORMAT_YVYU on Feb. 2014, and FFmpeg added AV_PIX_FMT_YVYU422 on Apr. 2014, which is probably why we didn't originally support it.